src/many-abci/src/main.rs

```diff
@@ -154,20 +154,26 @@ async fn main() {
         s.set_fallback_module(backend);
     }
 
-    let many_server = many::transport::http::HttpServer::new(server);
+    let mut many_server = many::transport::http::HttpServer::new(server);
+
+    signal_hook::flag::register(signal_hook::consts::SIGTERM, many_server.term_signal())
```

Comment:
nit: Sould we also add `SIGINT` (Ctrl-C)?

Reply:
Sure.